### ISSUE TITLE
Enhance chromosome name check function

### ISSUE THREAD

**Issue by GoogleCodeExporter** [shenlab-sinai/ngsplot#25]
```
Right now, it considers the chromosome names in a bam file to be either have 
"chr" or not at all. This has caused trouble for some people who have a mix of 
chromosome names in their bam files. A little flexibility is desired.
```

Original issue reported on code.google.com by `shenli.sam` on 31 Jul 2014 at 4:29

**Comment by jdblischak** [comment]
I agree this is limiting. In my case, I have the ERCC spike-ins in addition to the chromosomes. I'd prefer ngsplot to just ignore the ERCC chromosomes and reads mapping to them, unless of course I specify them with a custom bed file.

**Comment by lishen** [comment]
fixed by c8f1265309174fc7e440fa554948ed0faa6b1f32
